src/main/resources/view/AnnotationParagraphCard.fxml

```diff
@@ -9,24 +9,27 @@
 
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.TextField?>
```

Comment:
Not so important, but there are quite a few unused import statements here.

Reply:
Will update before mid-v1.4 (in case I still have to use them)